EcoData/R/data.R

```diff
@@ -355,6 +355,141 @@
+#'   \item{yellowCards}{number of yellow cards player received from referee}
+#'   \item{yellowReds}{number of yellow-red cards player received from referee}
+#'   \item{redCards}{number of red cards player received from referee}
+#'   \item{photoID}{ID of player photo (if available)}
+#'   \item{rater1}{skin rating of photo by rater 1 (5-point scale ranging from very light skin, to very dark skin)}
+#'   \item{rater2}{skin rating of photo by rater 2 (5-point scale ranging from very light skin, to very dark skin)}
+#'   \item{refNum}{unique referee ID number (referee name removed for anonymizing purposes)}
+#'   \item{refCountry}{unique referee country ID number (country name removed for anonymizing purposes)}
+#'   \item{Alpha_3}{unique referee country ID number (country name removed for anonymizing purposes)}
+#'   \item{meanIAT}{mean implicit bias score (using the race IAT) for referee country, higher values correspond to faster white | good, black | bad associations}
+#'   \item{nIAT}{sample size for race IAT in that particular country}
+#'   \item{seIAT}{standard error for mean estimate of race IAT }
+#'   \item{meanExp}{mean explicit bias score (using a racial thermometer task) for referee country, higher values correspond to greater feelings of warmth toward whites versus blacks}
+#'   \item{nExp}{sample size for explicit bias in that particular country}
+#'   \item{seExp}{standard error for mean estimate of explicit bias measure}
+#' }
+#' @author Florian Hartig
+#' @source \url{https://www.picostat.com/dataset/r-dataset-package-vcd-arthritis}
+"redCards"
 
 
 
+##
+#' plantPollinations
+#'
+#' plant pollination by different insects.
+#' This dataset is split and used for machine learning with an training and testing dataset, as well as the full dataset
+#' \docType{data}
+#' @format A 'data.frame':	20480 obs. of  18 variables:
+#' \describe{
+#'  \item{crop}{plant name}
+#'  \item{insect}{insect name}
+#'  \item{type}{type of plant (arboreous or herbaceous)}
+#'  \item{season}{flowering season}
+#'  \item{diameter}{flower diameter in mm}
+#'  \item{corolla}{corolla shape (open, campanulate, or tubular)}
+#'  \item{colour}{flower colour}
+#'  \item{nectar}{nectar (yes or no)}
+#'  \item{b.symstem}{ bloom system (type of pollination: insects, insects/wind, or insects/birds)}
+#'  \item{s.pollination}{self‐pollination (yes or no)}
+#'  \item{inflorescence}{inflorescence (yes, solitary, solitary/pairs, solitary/clusters)}
+#'  \item{composite}{composite flowers (yes or no)}
+#'  \item{guild}{five pollinator traits: guild (bumblebees, butterflies etc.)}
+#'  \item{tongue}{ tongue length in mm}
+#'  \item{body}{body size in mm}
+#'  \item{sociality}{soziality "Yes/NO"}
+#'  \item{feeding}{feeding behaviour (oligolectic, polylectic, or parasitic)}
+#'  \item{interaction}{Interaction between insect and plant(Yes/No)}
+#'}
+#' @author Florian Hartig
+#' @source \url{https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13329}
```

Comment:
You should also refer to https://figshare.com/articles/dataset/Global_pollinator_database/9980471/1